.github/workflows/mandatory-tests.yml

```diff
@@ -0,0 +1,11 @@
+name: Run mandatory tests
+on: pull_request
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install modules
+      run: npm install
+    - name: Run mandatory tests
+      run: npm test
```

Comment:
I think it would be good if npm test runs all tests, so it would just be a different command for the mandatory tests (will need to remember to change this when making that change)